.drone.yml

```diff
@@ -93,9 +76,10 @@ pipeline:
       status: failure
 
 matrix:
-  COMPOSER_BOUNDARY:
-    - lowest
-    - highest
-  PHP_VERSION:
-    - 8.0
-    - 8.1
+  include:
+    - CORE_VERSION: 9.4
```

Comment:
and then here we go for 9.4.0 and the `~` will install the latest stable or we could also do 9.4.* I think